### ISSUE TITLE
NotFoundError: driver_100_30frame/05251517_0433.MP4/00000.jpg

### ISSUE THREAD

**Issue by YilmazOrkun** [cardwing/Codes-for-Lane-Detection#62]
Hello, unfortunately I have problems with image paths. I already did read closed issues about that and the suggestion was to use absolute path. But where do I enter the absolute path? In the test.txt list file?

Thanks in advance!

**Comment by cardwing** [comment]
@YilmazOrkun, just print the path and check if it exists.

**Comment by YilmazOrkun** [comment]
Dear @cardwing, unfortunately I don't understand your answer. Could you please more specific.

My test.txt looks like this:

~/SCNN/data/CULane/driver_100_30frame/05251517_0433.MP4/00000.jpg
~/SCNN/data/CULane/driver_100_30frame/05251517_0433.MP4/00030.jpg
~/SCNN/data/CULane/driver_100_30frame/05251517_0433.MP4/00060.jpg

So I did use absolute path. And I call testing like this:

CUDA_VISIBLE_DEVICES="0" python tools/test_lanenet.py --weights_path ~/SCNN-Tensorflow/lane-detection-model/culane_lanenet_vgg_2018-12-01-14-38-37.ckpt-10000 --image_path ~/SCNN/data/CULane/list/test.txt --save_dir ~/SCNN-Tensorflow/lane-detection-model/predictions

But the images still can't be found and I get the error:
NotFoundError (see above for traceback): home/mrk/SCNN/data/CULane/driver_100_30frame/05251517_0433.MP4/00000.jpg

Thanks in advance

**Comment by rdevi-pz** [comment]
Hi. I am facing the same issue. I am trying to use image files of png type and I am facing the same notfound error though path is correct... 

PFB the code used: 
CUDA_VISIBLE_DEVICES="0" python tools/test_lanenet.py --weights_path /home/ec2-user/weights/model_culane-71-3/culane_lanenet_vgg_2018-12-01-14-38-37.ckpt-10000 --image_path /home/ec2-user/text_path/test_kitti.txt --save_dir result

PFB the error: 
NotFoundError (see above for traceback): home/ec2-user/2011_09_26/2011_09_26_drive_0002_sync/image_00/data/0000000000.png; No such file or directory
         [[node map/while/ReadFile (defined at /home/ec2-user/Codes-for-Lane-Detection/SCNN-Tensorflow/lane-detection-model/data_provider/lanenet_data_processor_test.py:57) ]]

PFB the result on going to the particular path: 
[ec2-user@ip-172-31-34-72 lane-detection-model]$ cd /home/ec2-user/2011_09_26/2011_09_26_drive_0002_sync/image_00/data/
[ec2-user@ip-172-31-34-72 data]$ ls
0000000000.png  0000000007.png  0000000014.png  0000000021.png  0000000028.png  0000000035.png  0000000042.png  0000000049.png  0000000056.png  0000000063.png  0000000070.png
0000000001.png  0000000008.png  0000000015.png  0000000022.png  0000000029.png  0000000036.png  0000000043.png  0000000050.png  0000000057.png  0000000064.png  0000000071.png
0000000002.png  0000000009.png  0000000016.png  0000000023.png  0000000030.png  0000000037.png  0000000044.png  0000000051.png  0000000058.png  0000000065.png  0000000072.png
0000000003.png  0000000010.png  0000000017.png  0000000024.png  0000000031.png  0000000038.png  0000000045.png  0000000052.png  0000000059.png  0000000066.png  0000000073.png
0000000004.png  0000000011.png  0000000018.png  0000000025.png  0000000032.png  0000000039.png  0000000046.png  0000000053.png  0000000060.png  0000000067.png  0000000074.png
0000000005.png  0000000012.png  0000000019.png  0000000026.png  0000000033.png  0000000040.png  0000000047.png  0000000054.png  0000000061.png  0000000068.png  0000000075.png
0000000006.png  0000000013.png  0000000020.png  0000000027.png  0000000034.png  0000000041.png  0000000048.png  0000000055.png  0000000062.png  0000000069.png  0000000076.png

**Comment by rdevi-pz** [comment]
Hi. I am facing the same issue. I am trying to use image files of png type and I am facing the same notfound error though path is correct...

PFB the code used:
CUDA_VISIBLE_DEVICES="0" python tools/test_lanenet.py --weights_path ~/weights/model_culane-71-3/culane_lanenet_vgg_2018-12-01-14-38-37.ckpt-10000 --image_path ~/text_path/test_kitti.txt --save_dir result

PFB the error:
NotFoundError (see above for traceback): ***************/2011_09_26/2011_09_26_drive_0002_sync/image_00/data/0000000000.png; No such file or directory
[[node map/while/ReadFile (defined at **************/Codes-for-Lane-Detection/SCNN-Tensorflow/lane-detection-model/data_provider/lanenet_data_processor_test.py:57) ]]

PFB the result on going to the particular path:
[************************ lane-detection-model]$ cd /home/ec2-user/2011_09_26/2011_09_26_drive_0002_sync/image_00/data/
[****************** data]$ ls
0000000000.png 0000000007.png 0000000014.png 0000000021.png 0000000028.png 0000000035.png 0000000042.png 0000000049.png 0000000056.png 0000000063.png 0000000070.png
0000000001.png 0000000008.png 0000000015.png 0000000022.png 0000000029.png 0000000036.png 0000000043.png 0000000050.png 0000000057.png 0000000064.png 0000000071.png
0000000002.png 0000000009.png 0000000016.png 0000000023.png 0000000030.png 0000000037.png 0000000044.png 0000000051.png 0000000058.png 0000000065.png 0000000072.png
0000000003.png 0000000010.png 0000000017.png 0000000024.png 0000000031.png 0000000038.png 0000000045.png 0000000052.png 0000000059.png 0000000066.png 0000000073.png
0000000004.png 0000000011.png 0000000018.png 0000000025.png 0000000032.png 0000000039.png 0000000046.png 0000000053.png 0000000060.png 0000000067.png 0000000074.png
0000000005.png 0000000012.png 0000000019.png 0000000026.png 0000000033.png 0000000040.png 0000000047.png 0000000054.png 0000000061.png 0000000068.png 0000000075.png
0000000006.png 0000000013.png 0000000020.png 0000000027.png 0000000034.png 0000000041.png 0000000048.png 0000000055.png 0000000062.png 0000000069.png 0000000076.png

**Comment by cardwing** [comment]
Can you `cd home/mrk/SCNN/data/CULane/driver_100_30frame/05251517_0433.MP4`? It should be '/home/...' instead of 'home/...'.
